src/screens/TripDetails/components/Time.tsx

```diff
@@ -36,15 +55,15 @@ const Time: React.FC<TimeValues> = (timeValues) => {
       );
     }
     case 'no-realtime': {
-      return (
-        <ThemeText testID="schCaTime">
-          <ThemeText>{t(dictionary.missingRealTimePrefix)}</ThemeText>
-          {scheduled}
-        </ThemeText>
-      );
+      return <ThemeText testID="schCaTime">{scheduled}</ThemeText>;
     }
     default: {
-      return <ThemeText testID="schTime">{scheduled}</ThemeText>;
+      return (
+        <View style={{flexDirection: 'row', alignItems: 'center'}}>
+          {showRealtime && realtimeIcon}
+          <ThemeText testID="schTime">{scheduled}</ThemeText>
```

Comment:
I know you haven't touched this, but I think we should show the `expected` time here. Even though `scheduled` and `expected` is pretty much the same in this situation, the `expected` time will be most correct, and might deviate with up to a minute from the `scheduled` time.